src/store/channels-list/saga.test.ts

```diff
@@ -20,17 +19,29 @@ import { AsyncListStatus } from '../normalized';
 import { conversationsChannel } from './channels';
 import { multicastChannel } from 'redux-saga';
 import { ConversationStatus, denormalize as denormalizeChannel } from '../channels';
-import { stubResponse } from '../../test/saga';
 import { StoreBuilder } from '../test/store';
 
-const MOCK_CHANNELS = [
-  { name: 'channel 1', id: 'channel_0001', url: 'channel_0001', icon: 'channel-icon', hasJoined: false },
-  { name: 'channel 2', id: 'channel_0002', url: 'channel_0002', icon: 'channel-icon', hasJoined: false },
-  { name: 'channel 3', id: 'channel_0003', url: 'channel_0003', icon: 'channel-icon', hasJoined: false },
-];
+const mockChannel = (id: string) => ({
+  id: `channel_${id}`,
+  name: `channel ${id}`,
+  icon: 'channel-icon',
+  hasJoined: false,
```

Comment:
This should probably have `isChannel: true` just to make sure it's really considered a Channel.